### DISCUSSION TITLE
newQueryWithoutScopes() not working

### DISCUSSION THREAD

**Discussion by prodixx** [Laravel-Backpack/community-forum#375] [Q&A (Help)] [answered]
I have a project (4.1) where i was using newQueryWithoutScopes() to clear the scopes and apply a simple filter. 

```
$this->crud->addFilter(
    [
        'type'  => 'simple',
        'name'  => 'archive',
        'label' => 'Arhiva'
    ],
    false,
    function ($value) {
        $this->crud->query = $this->crud->model->newQueryWithoutScopes();

        $this->crud->addClause('onlyArchived');
    }
);
```

After upgrading to backpack 5.4 im getting an error on CrudPanel\Traits\Query.php : 249

![image](https://user-images.githubusercontent.com/3352723/214097983-980ff94a-600c-4491-9493-9998d6447cf1.png)

```
PHP 8.1.13 (cli) (built: Nov 22 2022 15:49:14) (ZTS Visual C++ 2019 x64)

### LARAVEL VERSION:
v9.48.0@c78ae7aeb0cbcb1a205050d3592247ba07f5b711

### BACKPACK PACKAGE VERSIONS:
backpack/crud: 5.4.15
backpack/pro: 1.5.0
```

Has anyone else encountered this error?
Thanks

**Comment by pxpm** [comment, marked as answer]
Hum.. thanks for sharing @prodixx ! 

I think `newQueryWithoutScopes` also removes the `select *` that's present by default on every model query builder. 

Can you just confirm that changing `foreach($crudQuery->columns ?? [] as $column)` would fix this ? 

Would you like to send the PR for it ?

Cheers

**Comment by prodixx** [comment]
Yuup! That does the trick.

Yes, I will open a PR.

thank you.
